outputs.tf

```diff
@@ -2,3 +2,10 @@
 # Output
 # ------------------------------------------------------------------------------
 
+output "tunnel1_address" {
+  value = "${aws_vpn_connection.main.tunnel1_address}"
+}
+
+output "tunnel2_address" {
```

Comment:
Typo, should probably be `tunnel2_address` in the output here?

Reply:
Nice catch - fixed